src/js/actions/GroupsDataActions.js

```diff
@@ -53,11 +53,11 @@ export const findGroupDataItem = (contextId, groupData) => {
  * @description verifies that the data in the checkdata folder is reflected in the menu.
  * @return {object} action object.
  */
-export function verifyGroupDataMatchesWithFs() {
+export function verifyGroupDataMatchesWithFs(toolName) {
```

Comment:
would it be a good idea to add a default value to toolName? That way we avoid the undefined value.

verifyGroupDataMatchesWithFs(toolName = null)